dynamic-config.md

```diff
@@ -275,7 +279,7 @@ The following example shows how to modify `slow-threshold` online by using the `
 {{< copyable "sql" >}}
 
 ```sql
-set tidb_slow_log_threshold = 200;
+set `tidb_slow_log_threshold` = 200;
```

Comment:
This is valid syntax, but for consistency when setting a sysvar the docs currently don't escape the values. It is a slightly different case than `set config`, in that sysvars won't use any special characters that need escaping.

Reply:
@rleungx PTAL

Reply:
> in that sysvars won't use any special characters that need escaping

agree with @morgo

Reply:
```suggestion
set tidb_slow_log_threshold = 200;
```

Reply:
Done in https://github.com/pingcap/docs-cn/pull/6114